src/instruments/ecam-page-template/template.js

```diff
@@ -41,6 +41,19 @@
 
 // eslint-disable-next-line camelcase
 class A32NX_PAGE_NAME_Logic extends Airliners.EICASTemplateElement {
+    constructor() {
+        super();
+        // eslint-disable-next-line no-underscore-dangle
+        let lastTime = this._lastTime;
+        this.getDeltaTime = () => {
+            const nowTime = Date.now();
+            const deltaTime = nowTime - lastTime;
+            lastTime = nowTime;
```

Comment:
why redefine lastTime?

Reply:
Not sure, it's @beheh code.

Reply:
Because otherwise deltaTime will be 10, 20, 30, 40, 50 whereas it should be 10, 10, 10, 10, 10.